puppet/modules/jenkins_job_builder/manifests/init.pp

```diff
@@ -0,0 +1,68 @@
+# Class to install, configure, and maintain JJB, as well
+# as to deploy the actual jobs to jenkins
+#
+# Loosely based on
+# https://git.openstack.org/cgit/openstack-infra/puppet-jenkins/tree/manifests/job_builder.pp
+# and should probably be kept up to date from there
+#
+# $norun is our addition and is so named becuase noop is a bit too magic in puppet syntax...
+#
+class jenkins_job_builder (
+  $url = '',
+  $username = '',
+  $password = '',
+  $config_dir = '',
+  $norun = 'true',
+) {
+
+  if ! defined(Package['jenkins-job-builder']) {
+    package { 'jenkins-job-builder':
+      ensure   => present,
+      provider => 'pip',
+    }
+  }
+
+  file { '/etc/jenkins_jobs':
+    ensure => directory,
+  }
+
+  file { '/etc/jenkins_jobs/config':
+    ensure  => directory,
+    owner   => 'root',
+    group   => 'root',
+    mode    => '0755',
+    recurse => true,
+    purge   => true,
+    force   => true,
+    source  => $config_dir,
+    notify  => Exec['jenkins_jobs_update'],
+  }
+
+  # test for a string here since it's annoyingly hard to pass a boolean from foreman via yaml
+  if $norun == 'false' {
```

Comment:
Aren't these (test/update) the wrong way around?  Or am I mis-reading the double negative?